From: Albert Astals Cid <aacid@kde.org>
Date: Wed, 9 Nov 2016 23:36:36 +0100
Subject: Add library path so utempter binary is found in Ubuntu 16.10

REVIEW: 129361
---
 cmake/FindUTEMPTER.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/FindUTEMPTER.cmake b/cmake/FindUTEMPTER.cmake
index d3236c8..c5a1e41 100644
--- a/cmake/FindUTEMPTER.cmake
+++ b/cmake/FindUTEMPTER.cmake
@@ -40,6 +40,7 @@ find_file (UTEMPTER_EXECUTABLE utempter PATHS
     ${CMAKE_PREFIX_PATH}/libexec/utempter
     ${CMAKE_PREFIX_PATH}/lib/utempter
     /usr/libexec/utempter
+    /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}/utempter
 )
 
 # On FreeBSD for example we have to use ulog-helper
