diff --git a/cmake/FindUTEMPTER.cmake b/cmake/FindUTEMPTER.cmake
index 9773963..4921e58 100644
--- a/cmake/FindUTEMPTER.cmake
+++ b/cmake/FindUTEMPTER.cmake
@@ -32,9 +32,13 @@
 # SUCH DAMAGE.
 
 
-find_program (UTEMPTER_EXECUTABLE utempter PATHS
+# use find_file instead of find_program until this is fixed:
+# https://gitlab.kitware.com/cmake/cmake/issues/10468
+find_file (UTEMPTER_EXECUTABLE utempter PATHS
     ${KDE_INSTALL_FULL_LIBEXECDIR}/utempter
     ${KDE_INSTALL_FULL_LIBDIR}/utempter
+    ${CMAKE_INSTALL_PREFIX}/libexec/utempter
+    ${CMAKE_INSTALL_PREFIX}/lib/utempter
     /usr/libexec/utempter
 )
 
